index.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <title>Youtube</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+    <body>
+        <nav>
+            <div>
+                <image src="https://upload.wikimedia.org/wikipedia/commons/e/e1/Logo_of_YouTube_%282015-2017%29.svg" height= "100px" width="100px">
+            </div>
+
+            <div>
+                <input type="text" name="search" placeholder="search">
+            </div>
+            <div>
+                <button type="button" ><i class="fa fa-search"></i></button>
+            </div>
+
+            <div id="add-video-playlist">
+                <i class="fa fa-file-video-o" aria-hidden="true" style="font-size:24px"></i>
+            </div>
+
+            <div id="bell">
+                <i class="fa fa-bell" style="font-size:24px"></i>
+            </div>
+
+            <div id="profile-pic">
+                <img src="https://thumbor.forbes.com/thumbor/fit-in/416x416/filters%3Aformat%28jpg%29/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F593b2e4b31358e03e55a0e8c%2F0x0.jpg%3Fbackground%3D000000%26cropX1%3D634%26cropX2%3D2468%26cropY1%3D39%26cropY2%3D1874">
+            </div>
+
+            
+        </nav>
+        <section id="review-info">
+            <div>
```

Comment:
- [ ] **OPTIONAL** For good HTML semantics, the use of `<main>` tag is encouraged, it is used to mark the relevant part of the `<body>`